src/loaders/ccloudResourceLoader.ts

```diff
@@ -504,6 +504,28 @@ export class CCloudResourceLoader extends CachingResourceLoader<
 
     return resultRows;
   }
+
+  /**
+   * Returns a deduplicated list of provider/region pairs for all Flink compute pools
+   * across all environments the user has access to.
+   */
+  public async getComputePoolProviderRegions(): Promise<IProviderRegion[]> {
```

Comment:
Would it be easier if there was a similar `getDatabaseProviderRegions()` method, or even a generic helper function for anything that implemented `IProviderRegion`?